lua/nordic/init.lua

```diff
@@ -5,8 +5,27 @@ local M = {}
 
 local default_opts = {
     underline_option = 'none',
-    italic = true,
-    italic_comments = false,
+    italics = {
+        enabled = true,
```

Comment:
What do you think about this approach?

```lua
{
  italics = false -- or {} to disable italics
  italics = { 'attributes', 'comments', ...} -- to enable italics for named keywords
```

If someone doesn't define the italics key in his config it gets extended by the defaults anyway.

One drawback I see, with a list of strings is that if someone wants to add one keyword to the defaults he has to define all the defaults as well. In this case it would be nice to have an append method.

Reply:
> One drawback I see, with a list of strings is that if someone wants to add one keyword to the defaults he has to define all the defaults as well. In this case it would be nice to have an append method.

Indeed. Another alternative to having that `string: boolean` table could possibly be to have a callback that provides the list of defaults so that you can define like this:

```lua
-- No italics
italics = false

-- Just use defaults
italics = true

-- Custom list
italics = { 'a', 'b', 'c' }

-- With defaults
italics = function(def) return vim.tbl_extend({ 'a', 'b', 'c' }, def) end
```

This would have the benefit of consolidating things as well, but not sure the pattern is super intuitive :man_shrugging: 

But I'm open to your initial suggestion. Combined with the map loop it won't be as verbose as initially was.

Reply:
> One drawback I see, with a list of strings is that if someone wants to add one keyword to the defaults he has to define all the defaults as well. In this case it would be nice to have an append method.

This goes the other way as well. If someone wants to disable any of the defaults some definition would have to be added.

The more I think of it, the more I lean towards having a plain set of names :thinking: Another solution that came to mind for doing appending is having some kind of "magic" string (like `_` or `_defaults_` or whatever), but I don't like magical things in configurations :sweat_smile:

Reply:
I thought about a possible solution this weekend. What do you think about similar to https://github.com/folke/tokyonight.nvim#%EF%B8%8F-configuration?

Reply:
Sorry about late reply @delafthi, the notification slipped my mind.

Something like that would be nice, but I feel like #70 should come first because that would make things a bit nicer to work with in general (and very easy to manage backward compatability just as is here today).